assets/babel-plugin.js

```diff
@@ -9,8 +9,10 @@ function getWebDependencyName(dep) {
 }
 
 function getPackageVersion(package) {
-  const packageJsonPath = path.posix.join('./', 'node_modules', package, 'package.json');
-  const json = JSON.parse(fs.readFileSync(packageJsonPath));
+  // First three are path for @babel/cli, so ignore
+  const modulesPath = require.main.paths.slice(3);
```

Comment:
Because of `require.resolve`'s root module is `@babel/cli`, getting node_modules paths to find dependencies here.

Reply:
Oh right, great catch! Could you use `process.cwd()` instead? I think it will be more reliable / easier to explain behavior than basing it off of this `paths` property

Reply:
Nice, done!